cmd/bedPeOverlap/bedPeOverlap.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed/bedpe"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/interval"
+	"log"
+)
+
+func bedpeOverlap(bedpeSelectFile string, bedpeInFile string, contactOutFile string) {
+	var inIntervals = make([]interval.Interval, 0)
+	var leftOverlaps []interval.Interval
+	var rightOverlaps []interval.Interval
+	var rightHalf bedpe.BedPeHalf
+	var leftHalf bedpe.BedPeHalf
+	var err error
+
+	out := fileio.EasyCreate(contactOutFile)
+
+	contactRecords := bedpe.Read(bedpeSelectFile)
+
+	inBedPe := bedpe.Read(bedpeInFile)
+
+	for _, i := range inBedPe {
+		left, right := bedpe.SplitBedPe(i)
```

Comment:
i would declare left and right before the for loop. A line like 'var left, right bedpe.BedPeHalf' and then change the colon equals to just an equals. This will be a slight runtime optimization, as you won't need to reallocate memory at each iteration of the loop